test/built-ins/Number/S9.3.1_A2.js

```diff
@@ -150,140 +150,131 @@ if (Number("\u1680") !== 0) {
-  }	
+    $ERROR('#22.2: Number("\\u2005") === +0. Actual: -0');
+  }
 }
 
-// CHECK#24
+// CHECK#23
 if (Number("\u2006") !== 0) {
-  $ERROR('#24.1: Number("\\u2006") === 0. Actual: ' + (Number("\u2006")));
+  $ERROR('#23.1: Number("\\u2006") === 0. Actual: ' + (Number("\u2006")));
 } else {
   if (1/Number("\u2006") !== Number.POSITIVE_INFINITY) {
-    $ERROR('#24.2: Number("\\u2006") === +0. Actual: -0');
-  }	
+    $ERROR('#23.2: Number("\\u2006") === +0. Actual: -0');
+  }
 }
 
-// CHECK#25
+// CHECK#24
 if (Number("\u2007") !== 0) {
-  $ERROR('#25.1: Number("\\u2007") === 0. Actual: ' + (Number("\u2007")));
+  $ERROR('#24.1: Number("\\u2007") === 0. Actual: ' + (Number("\u2007")));
 } else {
   if (1/Number("\u2007") !== Number.POSITIVE_INFINITY) {
-    $ERROR('#25.2: Number("\\u2007") === +0. Actual: -0');
-  }	
+    $ERROR('#24.2: Number("\\u2007") === +0. Actual: -0');
+  }
 }
 
-// CHECK#26
+// CHECK#25
 if (Number("\u2008") !== 0) {
-  $ERROR('#26.1: Number("\\u2008") === 0. Actual: ' + (Number("\u2008")));
+  $ERROR('#25.1: Number("\\u2008") === 0. Actual: ' + (Number("\u2008")));
 } else {
   if (1/Number("\u2008") !== Number.POSITIVE_INFINITY) {
-    $ERROR('#26.2: Number("\\u2008") === +0. Actual: -0');
-  }	
+    $ERROR('#25.2: Number("\\u2008") === +0. Actual: -0');
+  }
 }
 
-// CHECK#27
+// CHECK#26
 if (Number("\u2009") !== 0) {
-  $ERROR('#27.1: Number("\\u2009") === 0. Actual: ' + (Number("\u2009")));
+  $ERROR('#26.1: Number("\\u2009") === 0. Actual: ' + (Number("\u2009")));
 } else {
   if (1/Number("\u2009") !== Number.POSITIVE_INFINITY) {
-    $ERROR('#27.2: Number("\\u2009") === +0. Actual: -0');
-  }	
+    $ERROR('#26.2: Number("\\u2009") === +0. Actual: -0');
+  }
 }
 
-// CHECK#28
+// CHECK#27
 if (Number("\u200A") !== 0) {
-  $ERROR('#28.1: Number("\\u200A") === 0. Actual: ' + (Number("\u200A")));
+  $ERROR('#27.1: Number("\\u200A") === 0. Actual: ' + (Number("\u200A")));
 } else {
   if (1/Number("\u200A") !== Number.POSITIVE_INFINITY) {
-    $ERROR('#28.2: Number("\\u200A") === +0. Actual: -0');
-  }	
+    $ERROR('#27.2: Number("\\u200A") === +0. Actual: -0');
+  }
 }
 
-// CHECK#29
+// CHECK#28
 if (Number("\u202F") !== 0) {
-  $ERROR('#29.1: Number("\\u202F") === 0. Actual: ' + (Number("\u202F")));
+  $ERROR('#28.1: Number("\\u202F") === 0. Actual: ' + (Number("\u202F")));
 } else {
   if (1/Number("\u202F") !== Number.POSITIVE_INFINITY) {
-    $ERROR('#29.2: Number("\\u202F") === +0. Actual: -0');
-  }	
+    $ERROR('#28.2: Number("\\u202F") === +0. Actual: -0');
+  }
 }
 
-// CHECK#30
+// CHECK#29
 if (Number("\u205F") !== 0) {
-  $ERROR('#30.1: Number("\\u205F") === 0. Actual: ' + (Number("\u205F")));
+  $ERROR('#29.1: Number("\\u205F") === 0. Actual: ' + (Number("\u205F")));
 } else {
   if (1/Number("\u205F") !== Number.POSITIVE_INFINITY) {
-    $ERROR('#30.2: Number("\\u205F") === +0. Actual: -0');
-  }	
+    $ERROR('#29.2: Number("\\u205F") === +0. Actual: -0');
+  }
 }
 
-// CHECK#31
+// CHECK#30
```

Comment:
I'm sorry you had to do this. We should look in to removing these unmaintainable patterns (for another day, of course).